request_context/src/main/java/uk/gov/dhsc/htbhf/requestcontext/RequestContextConfiguration.java

```diff
@@ -28,4 +29,17 @@ public RequestContext requestContext() {
     public RequestIdFilter requestIdFilter() {
         return new RequestIdFilter(requestContext(), mdcWrapper());
     }
+
+    @Bean
+    public RestTemplate restTemplateWithIdHeaders() {
```

Comment:
We should set the name annotation rather than rely on the method name, which will only work if the code is complied with debug flags on.